psscriptanalyzer/Backup-Database.psm1

```
@@ -0,0 +1,40 @@
+    Write-Host "removed $target"
+}
+
+function Process-Backup {
+    param(
+        [string]$Database,
+        [string]$Username,
+        [string]$Password
+    )
+    $timestamp = Get-Date -Format "yyyyMMdd-HHmmss"
+    $backupFile = "$Database-$timestamp.bak"
+    $target = Join-Path $Global:BackupRoot (Join-Path $Database $backupFile)
```

PSScriptAnalyzer (1.25.0)

[warning] Found global variable 'Global:BackupRoot'.

(PSAvoidGlobalVars)